cli/src/cli.rs

```diff
@@ -109,6 +109,7 @@ pub enum CliCommand {
     CreateNonceAccount {
         nonce_account: KeypairEq,
         lamports: u64,
+        authorized_noncer: Option<Pubkey>,
```

Comment:
`authorized_noncer` is kinda comical, perhaps `authorized_user` would be more approachable to a future reader that wasn't in the room when the Noncer joke was first cracked?

Reply:
The humor was what I used to justify the long name :laughing:

Reply:
I personally like it.   I sadly feel it'll get lost in translation though

Reply:
Now I need to know, and for posterity, what was the joke?

Reply:
You didn't miss much.  A Stake account has a Staker so a Nonce account should have a Noncer.   
😆😆😆😆 Get it?  😆😆😆😆

Reply:
👎

Reply:
Is it a reference to the urban-dictionary definition of nonce? Or...?

Reply:
OMG no!

Reply:
Oh, good. I don't get it, then, but 🤷‍♀

Reply:
Dear god...

Nonce = Number used ONCE !

Reply:
Please take this to Discord's terminology channel before this feature (not PR) ships. Names like this suggest the implementation may only be good enough for only whomever is asking for some consequence of it.  The name should stand on its own. Once pinned down, the terms should go into the book, preferably BEFORE the implementation.